nifi-framework-bundle/nifi-framework/nifi-administration/src/main/java/org/apache/nifi/admin/service/EntityStoreAuditService.java

```diff
@@ -119,6 +129,11 @@ public void addActions(final Collection<Action> actions) {
             }
             logger.debug("Actions added [{}]", actions.size());
         });
+        flowActionReporter.reportFlowActions(
+            actions.stream()
+                .map(actionConverter::convert)
+                .toList()
+        );
```

Comment:
Since the `FlowActionReporter` is a new optional feature, it seems best to avoid this call if it is not configured. That avoids the action conversion. Allowing `FlowActionReporter` to be `null` seems like a reasonable way to avoid the conversion, and the null check could be added before calling `reportFlowActions`